docs/UserGuide.md

```diff
@@ -90,7 +90,7 @@ The `Code Panel` allows users to see the code attributed to a specific author. C
 * Clicking the first icon beside the file title opens the history view of the file on github.
 * Clicking the second icon beside the file title opens the blame view of the file on github.
 * Code attributed to the author is highlighted in green.
-* Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the `...` icon.
+* Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the :heavy_plus_sign: icon.
```

Comment:
The plus sign does not correspond to the actual icon in the code view.

Reply:
This is the closest I can find without having to import third party libraries to use the actual HTML icon.